tests/test_ops_batchnorm.py

```diff
@@ -19,13 +19,15 @@
 import numpy as np
 
 from tests.utils import run_node
+import pytest
```

Comment:
Just a small suggestion, it would be nice to keep imports in a consistent order.

Reply:
Right, changed ;)